portfolio/src/main/webapp/movies.css

```diff
@@ -89,8 +89,92 @@ body {
     padding: 10px;
 }
 
+#btns-container {
+    display: flex;
```

Comment:
Nit: for CSS rules, put them in alphabetical order. 

ie
border:
border-color:
.
.
.
padding: 
etc. 

Please fix throughout.